assets/dev/js/editor/elements/views/behaviors/sortable.js

```diff
@@ -113,18 +113,22 @@ SortableBehavior = Marionette.Behavior.extend( {
 	},
 
 	// On sorting element
-	updateSort: function( ui ) {
+	updateSort: function( ui, newIndex ) {
+		if ( undefined === newIndex ) {
```

Comment:
Explain the situation in the code.

Reply:
Added an explanation comment above the function that used when there isn't a natural index